towhee/types/audio_frame.py

```diff
@@ -24,11 +24,12 @@ class AudioFrame(np.ndarray):
             The audio sample rate in Hz.
     """
 
-    def __new__(cls, data: np.ndarray, sample_rate: int = None):
+    def __new__(cls, data: np.ndarray, sample_rate: int = None, timestamp: int = None):
         # Cast `np.ndarray` to be `AudioFrame`.
         # See https://numpy.org/doc/stable/user/basics.subclassing.html for details.
         obj = np.asarray(data).view(cls)
         obj._sample_rate = sample_rate
+        obj._timestamp = timestamp
```

Comment:
can we just declare `timestamp` as a public attribute?

Reply:
`timestamp` shouldn't be changed after init, so use a `@property` function to get `timestamp`